SensateIoT.SmartEnergy.Dsmr.Parser.Common/Models/Telegram.cs

```diff
@@ -42,6 +42,7 @@ public class Telegram
         public decimal GasConsumption { get; set; } = 0M;
         [Obis("0-1:24.2.1", 0), TypeConverter(typeof(ObisTimestampConverter))]
         public DateTime GasTimestamp { get; set; }
+		public bool HasGasConsumption { get; set; }

```

Comment:
Fix indentation

Reply:
Fixed. Pulled the indentation type of this line check with the rest of the file.